src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitChoiceGroup.razor.cs

```diff
@@ -51,6 +53,9 @@ public bool IsRequired
             }
         }
 
+        /// <summary>
+        /// Callback called when SelectedKey parameter is changed.
+        /// </summary>
         [Parameter] public EventCallback<string?> SelectedKeyChanged { get; set; }
```

Comment:
again, please remove this documentation comment.
it is not correct for these specific parameters.